website/integrations/services/sharepoint-se/index.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Microsoft SharePoint Server Subscription Edition
```

Comment:
Can we make this title a bit shorter by removing "Subscription Edition" and instead add it to the first header below, on line 7?

Reply:
Thanks for your review and updates.
That the official name, however we can use it’s acronym we’ll know on the market “SE”